Project/src/main/resources/static/js/reviewModal.js

```diff
@@ -0,0 +1,9 @@
+$( document ).ready(function() {
+    $('#reviewModal').on('show.bs.modal', function (event) {
+        var button = $(event.relatedTarget)
+        var articleId = button.data('id')
+        console.log(articleId)
```

Comment:
console.log should not be in production code

Reply:
that code review thing you added actually catches that so he would've had to fix it anyway